services/src/integ-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceIntTest.java

```diff
@@ -65,14 +64,15 @@
 import java.util.Set;
 
 import static org.junit.Assert.assertTrue;
```

Comment:
Change to JUnit5 version (org.junit.jupiter.api.Assertions.assertTrue).  Note that you'll need to swap the order of the parameters where it's used so that the boolean is first.